Cancel selection with right click
diff --git a/crud.c b/crud.c
index 6e97a0a..2a86cc3 100644
--- a/crud.c
+++ b/crud.c
@@ -189,6 +189,8 @@ int main(int argc, char **argv) {
     XNextEvent(display, &event);
     switch (event.type) {
     case ButtonPress:
+      if (event.xbutton.button == Button3)
+        return 1;
       button_state = true;
       x = start_x = event.xbutton.x_root;
       y = start_y = event.xbutton.y_root;
